docs/python_support/quickstart.md

```diff
@@ -0,0 +1,222 @@
+# Python support in OVMS - quickstart {#ovms_docs_python_support_quickstart}
+
+OpenVINO Model Server allows users to write custom processing nodes in Python, so they may have full control over what happens with the data reaching such node and what comes out of it.
+
+In this quickstart you'll create a servable with a single custom Python code that will expect a string and return the same string, but all in uppercase. 
+
+To acheive that let's follow the steps:
+1. Prepare Docker Image And Workspace
+2. Create Client Application
+3. Write Python Code For The Server 
+4. Prepare Graph Configuration File
+5. Prepare Server Configuration File
+6. Deploy OpenVINO Model Server
+7. Send Requests From The Client
+
+### Step 1: Prepare Docker Image And Workspace
+
+Let's have all the work done in a new `workspace` directory. Also let's  the following subdirectories:
+- `workspace/models` (catalog that will be mounted to the deployment)
+- `workspace/models/python_model` (catalog with Python servable specification)
+
+You can do that in one go:
+
+```bash
+mkdir -p workspace/models/python_model && cd workspace
+```
+
+Since changing all the letters in the string to uppercase is a very basic use case, the basic Python-enabled model server image without extra Python packages is sufficient. If you'd need some external modules, you would need specifically add them to the image. 
+Until the image is published in next release, the image needs to be built from sources:
+
+```bash
+git clone https://github.com/openvinotoolkit/model_server.git
+cd model_server
+make docker_build MEDIAPIPE_DISABLE=0 PYTHON_DISABLE=0 OV_USE_BINARY=1 RUN_TESTS=0
+cd ..
+```
+
+You'll also need a client module so in your environment install a required dependency:
+```bash
+pip3 install tritonclient[grpc]
+```
+
+### Step 2: Create Client Application
+
+Server provides the data to your Python node as-is, so in the server side Python code you can expect an untouched data buffer and some metadata from the request. This means that your client and server code need to be complementary.
```

Comment:
At first it was had for me to understand the first sentence. Maybe explain buffer protocol?

Reply:
I wanted to hide the details at this point. Is it better now?